ccc/deliverydb.py

```diff
@@ -1,18 +1,32 @@
-from deliverydb.db import DeliveryDB
+import typing
+
+import psycopg
+
+from deliverydb.db import DeliveryDB, delivery_db_no_orm
 import model.compliancedb
 
 
 def default_with_db_cfg(
     db_cfg: model.compliancedb.ComplianceDbConfig,
     hostname: str = None,
-) -> DeliveryDB:
+    orm: bool = True,
+) -> typing.Union[DeliveryDB, psycopg.Connection]:
 
     if not hostname:
         hostname = db_cfg.hostname()
 
-    return DeliveryDB(
-        username=db_cfg.credentials().username(),
-        password=db_cfg.credentials().password(),
-        hostname=hostname,
-        port=db_cfg.port(),
-    )
+    if orm:
+        return DeliveryDB(
```

Comment:
I know, not part of this PR: I think, DeliveryDB should not deal w/ setting up sqlalchemy. It should rather be passed an initialised instance of a sqlalchemy-session(-factory), and expose (as it already does) methods that _use_ said session / session-factory